yelp_package/Makefile

```diff
@@ -41,13 +51,20 @@ build_trusty_docker:
 	cd dockerfiles/trusty/ && docker build -t "paastatools_trusty_container" .
 package_trusty: build_trusty_docker
 	cp ../requirements.txt ../requirements.txt.old
-	$(ADD_MISSING_DEPS)
+	cp ../debian/changelog ../debian/changelog.old
```

Comment:
These are going to start to get a bit confusing. Can we at least add a comment explaining that the old versions are backups so that we can temporarily add a `yelp1` changelog entry and some new requirements without leaving the modifications behind?